pkg/tpl/funcs.go

```diff
@@ -43,7 +43,7 @@ func GetSchemaLink(ref string) string {
 	idx := strings.LastIndex(ref, "/")
 	refName := ref[idx+1:]
 	linkPath := ref[:idx]
-	linkPath = strings.ReplaceAll(linkPath, "#", fmt.Sprintf("%s%s", "{{site.baseurl}}", referenceURL))
+	linkPath = strings.ReplaceAll(linkPath, "#", fmt.Sprintf("%s%s/reference/", "{{site.baseurl}}", referenceURL))
```

Comment:
I'm not entirely sure about this, I know there has been some effort on being able to use a folder different than `/reference` to put the end files in. @JayanSmart could you help us here?